src/js/controllers/preferencesExternal.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+angular.module('copayApp.controllers').controller('preferencesExternalController', function($scope, $stateParams, lodash, gettextCatalog, popupService, profileService, walletService) {
+  var wallet = profileService.getWallet($stateParams.walletId);
+  $scope.wallet = wallet;
```

Comment:
Oh. Thought you were referring to the whole file ;-)
Yes, if it's not referenced it should be removed.